packages/manager/modules/vrack/src/dashboard/vrack.constant.js

```diff
@@ -26,13 +26,28 @@ export const VRACK_URLS = {
   },
 };
 
+/**
+ * OVH services names as constants
+ * @type {string}
+ */
+export const IP_SERVICE = 'ip';
+export const IPLB_SERVICE = 'ipLoadbalancing';
+export const DEDICATED_SERVICE = 'dedicatedServer';
+export const DSI_SERVICE = 'dedicatedServerInterface';
+export const DCLOUDDC_SERVICE = 'dedicatedCloudDatacenter';
+export const DCLOUD_SERVICE = 'dedicatedCloud';
+export const OVHCC_SERVICE = 'ovhCloudConnect';
+export const CP_SERVICE = 'cloudProject';
+export const LVRACK_SERVICE = 'legacyVrack';
+export const MBM_SERVICE = 'managedBareMetal';
```

Comment:
It would be more efficient if you use only one object.

Reply:
made it inside one object